app/views/govuk_component/docs/search.yml

```diff
@@ -1,19 +1,22 @@
 name: "Search"
 description: "Search box"
+body: |
+  Component built primarily for the GOV.UK homepage (where it uses the on_dark_background option) but intended for use throughout the site.
+
+  Component provides only the search box itself and must be wrapped in a form.
 fixtures:
   default: {}
-  for_use_on_a_dark_but_not_black_background:
-    on_dark_background: true
-  change_label_text:
-    label_text: "Search all the things"
-  stop_the_label_appearing_inline:
+  apply_custom_styling_to_the_label_and_set_search_value:
     inline_label: false
-  apply_custom_styling_to_the_label:
+    label_text: "<span class=\"fixture-title\">Search results for <span class=\"visuallyhidden\">\"driving licence\"</span></span>"
```

Comment:
Still using the `fixture-title` class which will not give an accurate representation of the component (as the guide styles this class).

`apply_custom_styling_to_the_label` – this component is not isolated.

I expected a fixture like the following, with the h1 styles still in the component:
```yml
search_results_with_h1_in_label:
  label_text: "Search results for <h1 class=\"visuallyhidden\">\"driving licence\"</span></h1>"
  value: "driving licence"
```